com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Artistic/Adjustment/ReplaceColorNode.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class ReplaceColorNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,
+            "ReplaceColor",
+@"
+{
+    Distance = distance(From, In);
+    Out = lerp(To, In, saturate((Distance - Range) / max(Fuzziness, 1e-5f)));
+}",
+            new ParameterDescriptor("In", TYPE.Vec3, Usage.In),
+            new ParameterDescriptor("From", TYPE.Vector, Usage.In),//TODO: Need to be color picker 
+            new ParameterDescriptor("To", TYPE.Vector, Usage.In),//Need to be color picker 
```

Comment:
From and To need to be Vector3 data types.

Reply:
Fixed